### ISSUE TITLE
Problem with detecting imports (plat-linux on arm)

### ISSUE THREAD

**Issue by auuron** [Nuitka/Nuitka#249]
Versions:

0.6.1.1
Python: 3.5.3 (default, Jun  7 2018, 20:26:48) 
Executable: /home/root/venv/bin/python
OS: Linux
Arch: aarch64

Nuitka was instaled via pip in virtualenv (but i had no issues on the other platform/python). 

python -m nuitka --follow-imports --standalone my_app.py

```
Nuitka:WARNING:There is a problem with detecting imports, CPython said:
...
...
...
b'# /home/root/venv/lib/python3.5/plat-linux/__pycache__/IN.cpython-35.pyc matches /home/root/venv/lib/python3.5/plat-linux/IN.py'
b"# code object from '/home/root/venv/lib/python3.5/plat-linux/__pycache__/IN.cpython-35.pyc'"
b'Traceback (most recent call last):'
b'  File "/tmp/tmprgqqtoq_", line 20, in <module>'
b'    __import__(imp)'
b'  File "<frozen importlib._bootstrap>", line 969, in _find_and_load'
b'  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked'
b'  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked'
b'  File "<frozen importlib._bootstrap_external>", line 673, in exec_module'
b'  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed'
b'  File "/home/root/venv/lib/python3.5/plat-linux/IN.py", line 133, in <module>'
b'    __MHWORDSIZE = __WORDSIZE'
b"NameError: name '__WORDSIZE' is not defined"
b'# clear builtins._'
b'# clear sys.path'
b'# clear sys.argv'
b'# clear sys.ps1'
b'# clear sys.ps2'
...
...
```

**Comment by kayhayen** [comment]
These files in `plat-*` seem generated from system header files. I couldn't really find how or where they are used. So removing them, and I believe this must only be an optimization for Nuitka. Quite obviously the file is totally unused on your system. It might be used or may once have been for some `distutils` magic.

**Comment by kayhayen** [comment]
There is code on the factory branch that handles this. If you want to check it out, go here:

http://nuitka.net/doc/factory.html

**Comment by auuron** [comment]
Thanks, i will!

**Comment by auuron** [comment]
Seems that it worked out. Thanks!

**Comment by kayhayen** [comment]
Released as 0.6.2
